pkg/flow/logging/logger_test.go

```diff
@@ -139,7 +222,14 @@ func TestLevels(t *testing.T) {
 		{
 			name: "gokit error level - prints",
 			logger: func(w io.Writer) (log.Logger, error) {
-				logger, err := logging.New(w, debugLevel())
+				o := debugLevel()
+				logger, err := logging.New(w, o)
+				if err != nil {
+					return nil, err
+				}
+				if err := logger.Update(o); err != nil {
+					return nil, err
+				}
```

Comment:
nit: to reduce redundancy you can add a function that takes in the options, creates the logger, updates it and returns it

Reply:
Good callout. I've amended